scss/main.scss

```diff
@@ -0,0 +1,299 @@
+@import  
+        './_fonts' ,
+        './_variables',
+        './header.scss',
+        './tipsSection.scss',
+        './sliderSection.scss',
+        './faqs.scss',
+        './footer.scss';
+
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body, html{
+    overflow-x: hidden;
+    scroll-behavior: smooth;
+}
+.wrapp{
+    max-width: 1440px;
+    min-width: 320px;
+    margin: 0 auto;
+    padding: 0 20px;
+}
+
+.no-scroll{
+    overflow: hidden;
+}
+
+@media screen and (max-width: 1150px) {
+    .slide-title{
+        font-size: 60px !important;
+        line-height: 90px !important;
+    }
+}
+
+@media screen and (min-width: 992px){
+    .header-title{
```

Comment:
you have `.header-title` in `header.scss` file - move media-queries to that block like I mentioned in comment above.

Also move all other styles from this file

Reply:
Done